app/build.gradle

```diff
@@ -128,6 +128,7 @@ dependencies {
     implementation "ch.acra:acra-mail:$acraVersion"
     implementation "ch.acra:acra-dialog:$acraVersion"
 
+    implementation 'com.github.jorgecastilloprz:fabprogresscircle:1.01@aar'
```

Comment:
Couldn't check in details, but didn't see that you mentioned the use of this library in the about text. Can yoh please add it?

Reply:
It has been added in the last commit, thank you !

Reply:
I guess it's me, but I don't see it. Can you open my eyes? :smile: I looked here:
https://github.com/Piwigo/Piwigo-Android/blob/a2d516b62e556393aa8b0fa9e7809f6c33246155/app/src/main/res/values/strings.xml#L84-L101

ok, to be honest: I believe you either committed it only locally or to a branch that is not yet merged.